Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,21 @@
-import React from 'react';
-import './Button.css';
+import React from "react";
+import "./Button.css";
+import Operate from "../operate";
+
+const NumberButton = props => {
+  if (Number.isInteger(props.numprop)) {
+    return (
+      <button onClick={() => Operate(props.numprop)} className="numbut">
```

Comment:
this is heckin clever!! you could also have just passed in which className each button is when you passed in which number/operator they are, but this works too!